apps/wing-console/console/server/src/utils/createRouter.ts

```diff
@@ -76,6 +76,11 @@ export interface FileLink {
   column?: number;
 }
 
+export interface RouterMeta {
+  action?: string;
+  resource?: string;
+}
```

Comment:
What about explicitly stating that this meta property is about analytics?

```suggestion
export interface RouterMeta {
  analytics?: {
    action: string;
    resource: string;
  };
}
```